plugins/auth/login.js

```diff
@@ -28,26 +29,46 @@ module.exports = config => ({
                 ));
             }
 
+            const pathNames = request.path.split('/');
+            const scmContext = pathNames[pathNames.indexOf('login') + 1];
+
+            // Redirect to the default login path if request path doesn't have a context
+            if (!scmContext) {
```

Comment:
It seems you want to process a request with no scmContext, but you can't. scmContext parameter cannot be optional because of the existing optional `web` parameter.
I think you can add a default login route by calling loginRoute with empty scmContext. 
https://github.com/screwdriver-cd/screwdriver/pull/648/files#diff-cc45542b37e76ced3e366e4aa20f512cR152